internal/command/import_test.go

```diff
@@ -21,7 +21,9 @@ import (
 )
 
 func TestImport(t *testing.T) {
-	defer testChdir(t, testFixturePath("import-provider-implicit"))()
+	td := t.TempDir()
+	testCopyDir(t, testFixturePath("import-provider-implicit"), td)
+	t.Chdir(td)
```

Comment:
Some diffs in this PR also change tests to copy test artefacts into a temporary directory, instead of using directories in the repo directly.